helper/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,7 @@
 name: helper
 author: Luck
 version: ${project.version}
+api-version: 1.13
```

Comment:
If you set to 1.14, the plugin will not be enabled in 1.13 servers.

Reply:
If it's set to 1.13, does that mean it won't be used on 1.12 servers?

I'm part of a large network which uses helper on both 1.12 and 1.8 servers. You'd be hard pressed to get that restriction passed the PR.

Reply:
Spigot 1.12.x and below doesn't care what the `api-version` is set to, but Spigot 1.13+ does. Currently the only way to get what you expect on all versions is to set it to `api-version: 1.13`, which ensures it will run on any version released so far.

Reply:
Good to know. I work mostly with 1.12.2 servers and below, so I'd never seen it, before. :p

Reply:
@FerusGrim https://www.spigotmc.org/wiki/plugin-yml/